1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,29 @@
+let content = document.querySelector('#content')
+let listOfBooksUlEl= document.createElement('UL')
+
 function readingList(books) {
-  // Write your code here...
+  for(let i=0; i<books.length; i++){
+
+    let newParEl = document.createElement('p');
+    newParEl.innerHTML=books[i].title + ' by '+ books[i].author;
+
+    let bookCoverEl = document.createElement('img')
+    bookCoverEl.src=books[i].bookCoverImage;
+    bookCoverEl.width='200'
+
+    let listOfBooksEl= document.createElement('LI')
+    if (books[i].alreadyRead){
+      listOfBooksEl.className= 'green'
+    }
```

Comment:
Standard convention is to write if-else statements like below
```
if (condition) {
   statement1
} else {
   statement2
}
```
Examples can be seen here in the MDN docs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else